src/main/app/src/assets/i18n/nl.json

```
@@ -213,6 +213,7 @@
   "gegevens.mail.afzenders": "Mailafzenders",
   "gegevens.mail.templates": "Mailtemplates",
   "gegevens.mail.ontvangstbevestiging": "Automatische ontvangstbevestiging",
+  "gegevens.mail.ontvangstbevestiging.hint": "Een automatische ontvangstbevestiging kan alleen verstuurd worden bij zaken die direct vanuit een formulier zijn aangemaakt, én waarbij het e-mailadres van de initiator bekend is.",
```

Strikt genomen niet waar. Een productaanvraag kan van een formulier komen maar dat hoeft niet. Een andere use case is bijv via een fax mbv (geloof ik) Kofax software. Ook daar zal een productaanvraag van worden gegenereerd en dus een automatische ontvangstbevestiging van worden verstuurd. ZAC weet in theorie wel waar een productaanvraag vandaan komt (`Naam van de bron` veld in een productaanvraag) maar dat hangt nu nog geen logica aan vast en dat zou ik ook zo houden. 
```suggestion
  "gegevens.mail.ontvangstbevestiging.hint": "Een automatische ontvangstbevestiging kan alleen verstuurd worden bij zaken die direct vanuit een productaanvraag (bijvoorbeeld vanuit een formulier) zijn aangemaakt, én waarbij het e-mailadres van de initiator bekend is.",
```